api/filters.py

```diff
@@ -0,0 +1,31 @@
+from django_filters import rest_framework as filters
+
+from core.models import Asset
+
+
+class AssetFilter(filters.FilterSet):
+    email = filters.CharFilter(
+        field_name='assigned_to__user__email',
+        lookup_expr='icontains', )
+    model_number = filters.CharFilter(
+        field_name='model_number__model_number',
+        lookup_expr='iexact',
+        method='filter_with_multiple_query_values')
+    asset_type = filters.CharFilter(
+        field_name='model_number__make_label__asset_type__asset_type',
+        lookup_expr='iexact',
+        method='filter_with_multiple_query_values')
+
+    def filter_model_number(self, queryset, name, value):
```

Comment:
This method is redundant, as the statements on L11 and L12 would result in the same queryset. Delete `filter_model_number()` and `filter_asset_type()` below.